### ISSUE TITLE
`false` is being inserted into formatted phone number, when no NDC is being used 

### ISSUE THREAD

**Issue by ioki-klaus** [floere/phony#487]
What I did
----------

One of our vendors requires us to send them phone numbers in the format: `+{cc}.{rest}` (i.e. `+49.123456789`). We have formatted the number with the following code:

```ruby
phone_number = '49123456789'
Phony.format(phone_number, format: '+%{cc}.%{ndc}%{local}', local_spaces: '')
# => "+49.123456789"
```

Everything fine so far.

What happened
-------------

Today, a user signed up with a danish phone number and our formatting code sent the following phone number to our vendor:

```ruby
phone_number = '4512121212'
Phony.format(phone_number, format: '+%{cc}.%{ndc}%{local}', local_spaces: '')
# => "+45.false12121212"
```

What I expected to happen and why
---------------------------------

We understand, that some countries like [Denmark](https://github.com/floere/phony/issues/64) and [Liechtenstein](https://github.com/floere/phony/issues/259) do not use NDC. And it is completely fine, that `Phony.split` inserts a `false`, but when formatting the number to a string, I would not expect to suddenly find a `false` inside my string, just because the NDC for that country is not applicable.

We currently work around by using:

```ruby
phone_number = '4512121212'
if !Phony.split(phone_number).second
  Phony.format(phone_number, format: '+%{cc}.%{local}', local_spaces: '')
else
  Phony.format(phone_number, format: '+%{cc}.%{ndc}%{local}', local_spaces: '')
end
```

but I do not really like that we have to split, check and then format.

Personal opinion: When using `format` and the country does not use an NDC, it should simply return an empty string there. That would be more correct, than suddenly finding a `false` in my phone number.

Or maybe are we missing a placeholder we can use in the `format` method, that will just give us the complete number ndc+local or just local when no ndc applies?

**Comment by floere** [comment]
@ioki-klaus Klaus, I wish I could tell you you're `false`, but it's not. Thanks for the detailed entry, your example is now an example in Phony (locally) and I'll look at it now.

**Comment by floere** [comment]
@ioki-klaus Fix released in 2.19.3 🚀 Closing for now. Please let me know if it works for you, and reopen if not.

**Comment by ioki-klaus** [comment]
Thx for the quick fix!
